calendar/client.bal

```diff
@@ -0,0 +1,261 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/log;
+
+# Calender Client Object.
+#
+# + httpClient - the HTTP Client
+@display {
+    label: "Microsoft Calendar Client",
+    iconPath: "logo.svg"
+}
+public client class Client {
+    http:Client httpClient;
+    Configuration config;
+
+    # Initializes the Outlook Calendar connector client.
+    #
+    # + config - Configurations required to initialize the `Client` endpoint
+    # + return - An error on failure of initialization or else `()`
+    public isolated function init(Configuration config) returns error? {
+        self.config = config;
+        http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig clientConfig = config.clientConfig;
+        http:ClientSecureSocket? socketConfig = config?.secureSocketConfig;
+        self.httpClient = check new (BASE_URL, {
+            auth: clientConfig,
+            secureSocket: socketConfig
+        });
+    }
+
+    # #############################################################################
+    # Operations on a Event resource
+    # The Event resource is the top-level object representing a event in outlook.
+    # #############################################################################
+
+    # Get event by proving the ID
+    # Get the properties and relationships of the specified event object.
+    # API doc : https://docs.microsoft.com/en-us/graph/api/event-get
+    #
+    # + eventId - ID of an event. Read-only.  
+    # + timeZone - Preferred Time Zone of the start and end time. (Default : `UTC`)
+    # + contentType - Preferred Content-Type of the body. Values : `text` or `text` (Default : `html`)
+    # + queryParams - Optional query parameters. This method support OData query parameters to customize the response. 
+    # It should be an array of type `string` in the format `<QUERY_PARAMETER_NAME>=<PARAMETER_VALUE>`
+    # **Note:** For more information about query parameters, refer here: 
+    # https://docs.microsoft.com/en-us/graph/query-parameters
+    # + return - a record `Event` if success. Else `error`.
```

Comment:
Guideline says 

When you have more than one returned items:
# + return - Generated string token,  an `auth:Error` occurred while generating the token, or else () if nothing is to be returned
# + return - `true` if authentication is successful, `false` otherwise, or else an `auth:Error` occurred while authenticating the credentials.

Reply:
Changed in all the places.
/Thanks